sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py

```diff
@@ -378,6 +378,10 @@ class SystemEventNames(str, Enum, metaclass=CaseInsensitiveEnumMeta):
 
     ResourceNotificationsHealthResourcesAvailabilityStatusChangedEventName = 'Microsoft.ResourceNotifications.HealthResources.AvailabilityStatusChanged'
 
+    ResourceNotificationsResourceManagementCreatedOrUpdatedEventName = 'Microsoft.ResourceNotifications.Resources.CreatedOrUpdated'
```

Comment:
is the `ResourceManagement` portion needed here ? Looking at the other patterns the name and value are largely 1:1.
`ResourceNotificationsResourceCreatedOrUpdatedEventName` still relays the same meaning

Reply:
This was discussed in the Api review as well and from what I remember the service team argued that it was required but can double check with Josh

Reply:
service team wanted to keep ResourceManagement